extensions/podman/packages/extension/src/utils/powershell.ts

```diff
@@ -32,21 +32,28 @@ export interface PowerShellClient {
 const PowerShell5Exe = 'powershell.exe';
 class PowerShell5Client implements PowerShellClient {
   async isVirtualMachineAvailable(): Promise<boolean> {
+    const telemetry: Record<string, unknown> = {};
     try {
       // set CurrentUICulture to force output in english
-      const { stdout: res } = await extensionApi.process.exec(
+      const res = await extensionApi.process.exec(
         PowerShell5Exe,
         [
           '[System.Console]::OutputEncoding = [System.Text.Encoding]::Unicode; (Get-WmiObject -Query "Select * from Win32_OptionalFeature where InstallState = \'1\'").Name | select-string VirtualMachinePlatform',
         ],
         { encoding: 'utf16le' },
       );
-      if (res.indexOf('VirtualMachinePlatform') >= 0) {
+      telemetry['command'] = res.command;
+      telemetry['stdout'] = res.stdout;
+      telemetry['stderr'] = res.stderr;
+      if (res.stdout.indexOf('VirtualMachinePlatform') >= 0) {
         return true;
       }
     } catch (err) {
       // ignore error, this means that VirtualMachinePlatform not enabled
+      telemetry['error'] = err;
     }
+    const telemetryLogger = extensionApi.env.createTelemetryLogger();
```

Comment:
Do we really need to re-create a telemetry logger? we already created one, should be nice to reuse it

https://github.com/podman-desktop/podman-desktop/blob/6e8fc0a385ab359ea187870a82783551714f9deb/extensions/podman/packages/extension/src/extension.ts#L1142